contracts/0.8.25/vaults/interfaces/IStakingVault.sol

```diff
@@ -27,27 +27,59 @@ interface IStakingVault {
         bytes32 depositDataRoot;
     }
 
+    struct GuardianSignature {
```

Comment:
We need unstructured signature here, because PDG will consume something other than ECDSA as a signature, e.g. pubkey. So we need `bytes signature` here.